scipy/stats/_multivariate.py

```diff
@@ -2696,9 +2633,8 @@ class invwishart_gen(wishart_gen):
     ----------
     .. [1] M.L. Eaton, "Multivariate Statistics: A Vector Space Approach",
            Wiley, 1983.
-    .. [2] M.C. Jones, "Generating Inverse Wishart Matrices", Communications
-           in Statistics - Simulation and Computation, vol. 14.2, pp.511-514,
-           1985.
+    .. [2] S.D. Axen, "Efficiently generating inverse-Wishart matrices and
```

Comment:
It doesn't hurt to have more references anyways. So better to add your document at the end of the list

Reply:
My concern is that, since the old reference is really just explaining that the efficient way to invert a Wishart matrix is directly from its Cholesky factored generated by the standard algorithm, it could be misleading if we reference this, since we never sample or invert a Wishart matrix now. But if you think it's still better to include, I'll leave it in.

Reply:
You can always explain the situation "instead of the standard algorithm given in [2], here [4] is followed such that inverse of Wishart matrix is directly sampled without any inversion" or whatever the case might be. I'm not sure if that's a correct sentence but the point is the explanation.